components/dialog/test/dialog-fullscreen.visual-diff.js

```diff
@@ -85,7 +85,7 @@ describe('d2l-dialog-fullscreen', () => {
 					await page.$eval('#dialogLong #bottom', (bottom) => {
 						bottom.scrollIntoView();
 					});
-					await visualDiff.screenshotAndCompare(page, this.test.fullTitle());
+					await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { captureBeyondViewport: false });
```

Comment:
@svanherk is a genius -- thanks! I'll look into un-skipping the other tests with this approach too.